readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https:amitpatiljc.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https:amitpatiljc.github.io/layout_moyo-header/report/html_report/)
```

Comment:
```suggestion
- [DEMO LINK](https://amitpatiljc.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://amitpatiljc.github.io/layout_moyo-header/report/html_report/)
```

`https:link` will actually not redirect, but instead add this address to the current address and try to go there

Reply:
okay i will add